jerry-core/ecma/base/ecma-helpers-value.c

```diff
@@ -564,6 +582,22 @@ ecma_get_object_from_value (ecma_value_t value) /**< ecma value */
 } /* ecma_get_object_from_value */
 
 /**
+ * Invert a boolean value
+ *
+ * @return ecma value
+ */
+inline ecma_value_t __attr_pure___ __attr_always_inline___
+ecma_invert_boolean_value (ecma_value_t value) /**< ecma value */
+{
+  JERRY_STATIC_ASSERT ((ECMA_SIMPLE_VALUE_FALSE | 0x1) == ECMA_SIMPLE_VALUE_TRUE
```

Comment:
OK, at least move out of the function. IMHO, it should be placed at the top of the file like in `ecma-helpers.c`